test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -44,13 +43,13 @@ public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
                 .Execute("new");
 
             new TestCommand("dotnet") { WorkingDirectory = rootPath }
-                .Execute("restore");
+                .Execute("restore3 -s https://dotnet.myget.org/F/cli-deps/api/v3/index.json /p:SkipInvalidConfigurations=true");
```

Comment:
The need to include CLI deps is point-in-time. Once the v1 packages are on NuGet we'll drop the -s parameter altogether.